### DISCUSSION TITLE
Money is disappeared

### DISCUSSION THREAD

**Discussion by benme11** [Chia-Network/chia-blockchain#8091] [Q&A]
I did not transfer out, but my money disappeared? And the maximum number of data only supports 50 . Why?

![11](https://user-images.githubusercontent.com/89021272/129578720-77840013-34dc-40e9-9ef9-07a877a34e10.jpg)
![22](https://user-images.githubusercontent.com/89021272/129578732-45fed7c1-c4f6-4392-9eef-0223152353b1.jpg)

**Comment by BrandtH22** [comment]
@benme11 , unfortunately it appears someone has gained access to your keyset and/or device and stolen your funds. Since chia is a truly decentralized network there is no way to reclaim the funds or revert the transaction(s). One thing we recommend is ensuring that you and your farm are following security best practices to reduce malicious actors ability to gain access to your keys and/or device, many of these best practices are outlined here: https://www.chia.net/2021/05/28/securing-your-chia-how-to-be-a-hard-target/

Since we unfortunately cannot do more to assist with the stolen funds we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
